src/asn1.mli

```diff
@@ -238,6 +278,14 @@ sig
   val dh_grammar : DH.Params.t Asn.t
 end
 
+module CVC :
```

Comment:
Is there a problem with the way it's currently structured, though? Because extracting separate `RSA.parse` and `ECDSA.parse` functions would be a bit tricky given the structure of the parser.